mitodl/mitxpro
# Add change password form

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #368 
Fixes https://github.com/mitodl/mitxpro/issues/439

#### What's this PR do?

- Update the user dropdown menu to include settings.
- Add a change password form to allow users to change their password.
- Rename old `ChangePasswordForm` which was used in forgot password flow, to `ResetPasswordForm`
 - Add new `ChangePasswordForm` to use actual change password flow.

#### How should this be manually tested?

- Click on user dropdown
- Click on settings or goto `/change-password` directly
- You should be able to change your password.

#### Screenshots (if appropriate)
**Success case**
![Screenshot 2019-10-22 at 6 10 48 PM](https://user-images.githubusercontent.com/4245618/67289236-b9093a80-f4f7-11e9-8773-61e2e9a0aabf.png)

**Failure**
![Screenshot 2019-10-22 at 6 11 04 PM](https://user-images.githubusercontent.com/4245618/67289238-b9a1d100-f4f7-11e9-938d-10687d07f91c.png)

**user dropdown**
<img width="340" alt="Screenshot 2019-10-22 at 6 13 33 PM" src="https://user-images.githubusercontent.com/4245618/67289239-b9a1d100-f4f7-11e9-9a2f-72e4fa5fac6d.png">
